src/GafferBindings/ValuePlugBinding.cpp

```diff
@@ -203,7 +221,7 @@ std::string ValuePlugSerialiser::repr( const Gaffer::ValuePlug *plug, const std:
 		}
 
 		const std::string defaultValue = valueRepr( pythonDefaultValue );
-		if( defaultValue.size() && defaultValue[0] != '<' )
+		if( defaultValue.size() )
```

Comment:
Out-of-scope for this PR, but there seems to be a mix of checking the `valueRepr` result or not. Eg: min/max and setValue serialisation just assume it works, but default, well guarded. Is that intentional?

Reply:
I noticed that too, but I don't know the history. Min/max should be irrelevant because they're limited to types where we already know `repr()` works. And I believe the rest is moot after this PR, because there should be no unserialisable values any more. I just left the default check there in case I was missing something.